backend/alembic/versions/f1e2d3c4b5a6_unify_share_tables.py

```
@@ -0,0 +1,338 @@
+        op.create_foreign_key(
+            'fk_resource_members_user_id',
+            'resource_members',
+            'users',
+            ['user_id'],
+            ['id'],
+        )
+
+
+def downgrade() -> None:
+    """Recreate legacy tables (without restoring data)."""
+    conn = op.get_bind()
```

Ruff (0.14.14)

[error] Local variable `conn` is assigned to but never used

Remove assignment to unused variable `conn`

(F841)